engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -655,6 +665,9 @@ void paladin_t::target_mitigation( school_e school,
     s -> result_amount *= 1.0 + buffs.blessing_of_dusk -> value();
   }
 
+  if ( buffs.devotion_aura -> up() )
+    s -> result_amount *= 1.0 + buffs.devotion_aura -> data().effectN( 1 ).percent();
```

Comment:
If you set devo aura's default value, you can just replace this with 
```suggestion
  s -> result_amount *= 1.0 + buffs.devotion_aura -> value();
```